src/core/iomgr/unix_sockets_posix_noop.c

```diff
@@ -35,7 +35,10 @@
 
 #ifndef GPR_HAVE_UNIX_SOCKET
 
-void grpc_create_socketpair_if_unix(int sv[2]) {}
+void grpc_create_socketpair_if_unix(int sv[2]) {
+  // TODO(ahedberg): Please fill in an implementation for non-Unix-sockets case
```

Comment:
Yes, sorry about that; I've tried to be more explicit in #5946 but I am not sure that that's the right suggestion. I'll change the TODO here as well.